sdcflows/workflows/apply/correction.py

```diff
@@ -26,7 +26,7 @@
 from niworkflows.engine.workflows import LiterateWorkflow as Workflow
 
 
-def init_unwarp_wf(omp_nthreads=1, debug=False, name="unwarp_wf"):
+def init_unwarp_wf(free_mem=None, omp_nthreads=1, debug=False, name="unwarp_wf"):
```

Comment:
just as a nitpick - since this function isn't keyword only, this could throw off any code using this positionally
```suggestion
def init_unwarp_wf(*, free_mem=None, omp_nthreads=1, debug=False, name="unwarp_wf"):
```